XouDevSpec/Classes/Model/ApiCallerModel.swift

```diff
@@ -28,10 +28,11 @@ public struct ApiCallerModel {
     
     
     public  func getAnimeData(completetionHandler: @escaping(Result<[AnimeDetails], AnimeError>) -> Void) {
+    
+        var request = URLRequest(url: NSURL(string:  "https://jikan1.p.rapidapi.com/search/anime?q=\(searchText)")! as URL,
```

Comment:
One suggestion is to have a different class for your URLs where they can be enums. Will support reusing them later

Reply:
That's not a bad idea thanks!